src/screens/VerificationScreen.js

```diff
@@ -1,50 +1,169 @@
 import * as React from 'react';
+import { useState, useRef } from 'react';
 
 import AsyncStorage from '@react-native-async-storage/async-storage';
 import firebase from 'firebase/compat/app';
 import { shape, func, string } from 'prop-types';
 import {
+  Dimensions,
+  Pressable,
+  TouchableWithoutFeedback,
   Text,
   View,
   TextInput,
   Button,
+  Keyboard,
   StyleSheet,
   TouchableOpacity,
+  KeyboardAvoidingView
 } from 'react-native';
 import { Title } from 'react-native-paper';
 
 import ViewContainer from '../components/ViewContainer';
+import { Inter_200ExtraLight } from '@expo-google-fonts/inter';
 
 const styles = StyleSheet.create({
-  separator: {
-    marginVertical: 30,
-    height: 1,
-    width: '80%',
-    backgroundColor: '#eee',
+  heading: {
+    fontSize: 28,
+    fontWeight: '700',
+    marginTop: '6%',
   },
+  caption: {
+    marginTop: '2.3%',
+    marginRight: 12,
+    marginLeft: 12,
+    textAlign: 'center',
+    fontSize: 17,
+    color: '#777',
+  },
+  captionPhoneNumber: {
+    width: '100%',
+    display: 'flex',
+    textAlign: 'right',
+    marginRight: 12,
+    marginLeft: 12,
+    fontSize: 17,
+    color: 'black',
+  },
+  rowContainer: {
+    flexDirection: 'row'
+  },
+  input: {
+    textAlign: 'center',
+    marginBottom: 15,
+    marginRight: 7,
+    marginLeft: 7,
+    width: '12.85%',
+    height: 50,
+    fontSize: 18,
+    position: 'relative',
+    borderWidth: 2,
+    borderColor: '#52bd41',
+    borderRadius: 8,
+  },
+  inputUnfocused: {
+    textAlign: 'center',
+    marginBottom: 15,
+    marginRight: 7,
+    marginLeft: 7,
+    width: 40,
+    height: 50,
+    fontSize: 18,
+    position: 'relative',
+    borderWidth: 2,
+    borderColor: '#ADB0B9',
+    borderRadius: 8,
+  },
+  containerForVerficationInput: {
+    padding: '2.3%',
+    backgroundColor: "#ffffff",
+    flexDirection: 'row'
+  },
+  codeResent: {
+    padding: '2.3%',
+    color: "#52bd41",
+  },
+  button: {
+    alignItems: 'center',
+    justifyContent: 'center',
+    paddingVertical: 15,
+    borderRadius: 8,
+    elevation: 3,
+    backgroundColor: '#52bd41',
+    position: 'relative',
+    width: '94%',
+    bottom: 0,
+    marginBottom: '3%',
+    marginLeft: '3%',
+
+  },
+  text: {
+    fontSize: 16,
+    lineHeight: 21,
+    fontWeight: 'bold',
+    letterSpacing: 0.25,
+    color: 'white',
+  },
+  backgroundWhite: {
+    backgroundColor: 'white',
+  },
+  startLeft: {
+    width: '100%',
+    alignItems: 'flex-start',
+  },
+  resendButton: {
+    color: 'black',
+    margin: '0%',
+    padding: '0%',
+    fontSize: 20,
+  }
 });
 
 export default function VerificationScreen({ route, navigation }) {
   const [message, showMessage] = React.useState({
     text: '',
   });
-  const { verificationId } = route.params;
-  const [verificationCode, setVerificationCode] = React.useState('');
-  return (
```

Comment:
Not sure where it is in the code, but verification screen has header at the top of the screen that says "Verify" with a back button. This elem is not in the fimga, confirm w/ Danielle if it should be present